### ISSUE TITLE
Replace the CSS sanitizer with something that parses CSS

### ISSUE THREAD

**Issue by gsnedders** [html5lib/html5lib-python#152]
Currently the sanitizer uses horrible regexp which I have no reason to believe correct whatsoever to parse CSS. We should move to a CSS parser, like https://github.com/SimonSapin/tinycss/.

**Comment by pzingg** [comment]
Also does not parse transform properties with multiple operations: 

width: 386.00px; height: 289.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);

**Comment by willkg** [comment]
@gsnedders Do you want help with this issue? I can test it out with Bleach and then create a PR for html5lib. Would that help?

**Comment by gsnedders** [comment]
@willkg Yes! Please feel free to. If you want a pointer, you probably want to start at `tinycss.CSS21Parser.parse_style_attr`.
